exercises/binary-search/src/example/java/BinarySearch.java

```diff
@@ -0,0 +1,40 @@
+
+import java.util.List;
+
+public class BinarySearch<T extends Comparable<T>> {
```

Comment:
Hello, and thank you for your comments. Here is a question: Do we need *.keep* files? Spotted that some were removed in one of the commits.